src/components/BooksList.js

```diff
@@ -0,0 +1,7 @@
+export default function BooksList() {
+  return (
+    <div>
+      <h2>No books added yet</h2>
+    </div>
+  );
```

Comment:
Books are required to be rendered on the page. Kindly add a book structure here and attempt to render them to the page.